docs/Release_Notes.gfs.v16.1.5.txt

```diff
@@ -0,0 +1,117 @@
+
+IMPLEMENTATION INSTRUCTIONS
+ 
+The NOAA VLab and both the NOAA-EMC and NCAR organization spaces on GitHub.com are used to manage the GFS.v16.1.5 code. The SPA(s) handling the GFS.v16.1.5 implementation need to have permissions to clone VLab gerrit repositories and the private NCAR UPP_GTG repository (the “nwprod” account has access to the GTG repository). All NOAA-EMC organization repositories are publicly readable and do not require access permissions. Please follow the following steps to install the package on WCOSS-Dell.
+ 
+While logged in under the “nwprod” account:
+
+1) cd $NWROOTp3
+2) mkdir gfs.v16.1.5
+3) cd gfs.v16.1.5
+4) git clone -b EMC-v16.1.5  https://github.com/NOAA-EMC/global-workflow.git .
+5) cd sorc
+6) ./checkout.sh -o
+   * This script extracts the following GFS components:
+     MODEL      tag GFS.v16.0.17			Jun.Wang@noaa.gov
+     GSI        tag gfsda.v16.1.5   	    		Russ.Treadon@noaa.gov
+     GLDAS     	tag gldas_gfsv16_release.v1.12.0     	Helin.Wei@noaa.gov
+     UFS_UTILS 	tag ops-gfsv16.0.0  	               	George.Gayno@noaa.gov
+     POST       tag upp_gfsv16_release.v1.1.4        	Wen.Meng@noaa.gov
+     WAFS       tag gfs_wafs.v6.0.22                  	Yali.Mao@noaa.gov
+
+7) ./build_all.sh
+   * This script compiles all GFS components. Runtime output from the build for each package is written to log files in directory logs. To build an individual program, for instance, gsi, use build_gsi.sh.
+8) ./link_fv3gfs.sh nco dell  
+
+SORC CHANGES
+
+* checkout.sh will checkout the following changed model tags:
+  * MODEL tag GFS.v16.0.17
+    * conf/configure.fv3.wcoss_dell_p3 and other platform configure.fv3 files: add “-g -traceback” to FFLAGS_OPT, CFLAGS_OPT, and CFLAGS_REPRO compilation settings
+  * GSI tag gfsda.v16.1.5
+    * src/gsi/genqsat.f90: add a lower bound of qsat to avoid unphysical saturation specific humidity to be generated.
+
+FIX CHANGES
+
+* fix/fix_gsi:
+  * new files:
+    * Rcov_iasicsea
+    * Rcov_iasicland
+  * modified files:
+    * global_satinfo.txt: Turn on assimilating Metop-C IASI data and stop monitoring CrIS_NPP MW channels (iuse changed from -1 to -2)
+    * global_anavinfo.l127.txt: Add correlated observation errors for Metop-C IASI data and remove correlated observation errors for Metop-A IASI
+    * Rcov_iasibsea: recomputed from recent operations
+    * Rcov_iasibland: recomputed from recent operations
+    * Rcov_crisn20: recomputed from recent operations
+    * Rcov_crisnpp: updated and the MW channels are deleted from Rcov
+    * gfsv16_historical/: Add fix files for retrospective parallels.  Does not impact operations.
+* fix_am/co2dat_4a:
+  * global_co2historicaldata_2020.txt
+  * global_co2historicaldata_2021.txt_proj_u
+  * global_co2historicaldata_2022.txt_proj
+* fix_am/fix_co2_proj:
+  * global_co2historicaldata_2022.txt
+* fix_am/fix_co2_update:
+  * global_co2historicaldata_2021.txt
+ 
+PARM/CONFIG CHANGES
+
+* No change from GFS v16.1.4
```

Comment:
Should we update the parm section since the file parm/config/config.anal was changed. The modification would not affect the operation since it only applies to the RUN_ENVIR == "emc" if blocks.

Reply:
Yes, good catch, perhaps something like this?

```
PARM/CONFIG CHANGES

* config.anal
  * update global_convinfo.txt notes and global_satinfo.txt if-blocks for retrospective dates
```

Please feel free to adjust that to what you'd prefer. I will commit the change, thanks!